### ISSUE TITLE
SafariDriver no longer works with 2.53

### ISSUE THREAD

**Issue by kurtharriger** [SeleniumHQ/selenium#2132]
## Meta -
OS:  OS X: 10.10.5
Selenium Version:   2.53
Browser:  Safari
Browser Version:    9.1.1 (10601.6.17)

## Expected Behavior -

Displays the Title was: Google

## Actual Behavior -

16:30:25.120 WARN - Exception: Failed to connect to SafariDriver after 10058 ms

## Steps to reproduce 

[See this github repo](https://github.com/kurtharriger/safari-driver-issue) for minimal example, logs and screenshots

**Comment by raghugitrepo** [comment]
Seeing the same issue but is in consistent. on below environment.
Setup failed:
WebDriverException: Message: Failed to connect to SafariDriver after 10058 ms
Build info: version: '2.53.0', revision: '35ae25b', time: '2016-03-15 17:00:58'
System info: host: ', ip: 'XXXXXX', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.10.5', java.version: '1.8.0_73'
Driver info: driver.version: SafariDriver
Stacktrace:
    at org.openqa.selenium.safari.SafariDriverCommandExecutor.start (SafariDriverCommandExecutor.java:121)
    at org.openqa.selenium.safari.SafariDriver.startClient (SafariDriver.java:76)
    at org.openqa.selenium.remote.RemoteWebDriver.<init> (RemoteWebDriver.java:119)
    at org.openqa.selenium.remote.RemoteWebDriver.<init> (RemoteWebDriver.java:144)
    at org.openqa.selenium.safari.SafariDriver.<init> (SafariDriver.java:62)
    at org.openqa.selenium.safari.SafariDriver.<init> (SafariDriver.java:53)
    at sun.reflect.GeneratedConstructorAccessor21.newInstance (None:-1)
    at sun.reflect.DelegatingConstructorAccessorImpl.newInstance (DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance (Constructor.java:422)
    at org.openqa.selenium.remote.server.DefaultDriverProvider.callConstructor (DefaultDriverProvider.java:103)
    at org.openqa.selenium.remote.server.DefaultDriverProvider.newInstance (DefaultDriverProvider.java:97)
    at org.openqa.selenium.remote.server.DefaultDriverFactory.newInstance (DefaultDriverFactory.java:60)
    at org.openqa.selenium.remote.server.DefaultSession$BrowserCreator.call (DefaultSession.java:222)
    at org.openqa.selenium.remote.server.DefaultSession$BrowserCreator.call (DefaultSession.java:1)
    at java.util.concurrent.FutureTask.run (FutureTask.java:266)
    at org.openqa.selenium.remote.server.DefaultSession$1.run (DefaultSession.java:176)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:617)
    at java.lang.Thread.run (Thread.java:745)
OS: OSX El capitan
Selenium Version: 2.53.0 
Browser: Safari
Browser Version: 9.1
SafariDriver Installed manually on safari browser.

**Comment by 5hawnknight** [comment]
@kurtharriger with the same configuration, I was able to get the expected behavior to work -- did you download your extension from here? -- http://selenium-release.storage.googleapis.com/index.html?path=2.48/

**Comment by kurtharriger** [comment]
2.48 sorta works but appears fail on some apis.  I then tried the latest version with 2.53 but it does not work at all.  http://selenium-release.storage.googleapis.com/2.53/SafariDriver.safariextz

**Comment by 5hawnknight** [comment]
your above link does not provide a valid location for the SafariDriver.safariextz
there is not a "newer" version of the SafariDriver.safariextz beyond 2.48
what do you mean by, "fail on some apis"?

**Comment by kurtharriger** [comment]
Sorry it was 2.48 not 2.53. In my github repo with the sample code I had copied a screenshot of the extensions page showing 2.48 installed.  I think I was initially having problems with 2.45 and updated to 2.48.  At the time I think 2.45 was documented as the latest release but poking around in downloads I found version 2.48 but it wasn't working so when I updated my docs I must have incorrectly updated my link from 2.45 to 2.53 instead of 2.48. 

I had removed the extension from safari, but now when I reinstalled it and tried to reproduce the error again I find that 2.48 is working perfectly now.  I'm not sure what has changed since, but tests are now magically passing in safari, so I'll close the issue.  

Thanks for looking into it.
